.coverage.sh

```diff
@@ -11,4 +11,9 @@ ici_exec_in_workspace "$extend" "$ws" catkin build robot_calibration -v --no-dep
 
 echo "Uploading coverage results to codecov.io"
 
+# Remove duplicated information
+rm "$ws/build/robot_calibration/*.cleaned"
+rm "$ws/build/robot_calibration/*.removed"
```

Comment:
I think we should do the same for python.
For whatever reason, if there was a .coverage.hostname.blah_blah file from earlier, it will get included in the "combine" call.

Reply:
the "python-coverage erase" gets called before tests get executed (as long as you use the ${pkg}_coverage_report target - which calls the erase)

Reply:
Even here, this isn't "old" data - it's from this run, but the files don't deleted after we filter the data

Reply:
but in my testing, erase wasn't erasing the .coverage.hostname.blah intermediate files. the ones that are created for each process.